cmd/kubeadm/app/cmd/init.go

```diff
@@ -233,14 +234,19 @@ func (i *Init) Run(out io.Writer) error {
 		return fmt.Errorf("couldn't parse kubernetes version %q: %v", i.cfg.KubernetesVersion, err)
 	}
 
-	// PHASE 1: Generate certificates
-	if err := cmdphases.CreatePKIAssets(i.cfg); err != nil {
-		return err
-	}
+	if !certphase.UsingExternalCA(i.cfg) {
```

Comment:
According to the issue, it is expected to have also required kubeconfig files on disk. Is kubeadm going to test this precondition?